app/forms/placement_form.rb

```diff
@@ -38,7 +44,33 @@ def title
   end
 
   def save!
-    # TODO: Implement persistence logic
-    false
+    return false unless valid?
+
+    if school_id.present? && (school = School.find_by(id: school_id)).present?
+      create_placement_from(school)
+    else
+      create_placement_from(
+        School.create!(new_school_attributes.merge(lead_school: false)),
```

Comment:
I'm not sure this is correct. I think the `schools` table stores a list of schools exclusively imported from the GIAS script. 

I think we want to create a `@trainee.placements.create(new_attributes)` here and set `name`, `urn`, `postcode` on that table since it could be anything. This is a join table and could have a school associated (it's optional) if one has been found from a lookup otherwise it should have arbitrary information about a placement given by the provider.